### PR TITLE
[TT-6670] Add OAS mock response schema

### PR DESCRIPTION
**Pull request by furkansenharputlu**
This PR adds schema for mock response middleware in OAS definition.

### CODE REVIEW

**Review by lghiur — APPROVED** [review]

File: apidef/oas/schema/x-tyk-api-gateway.json [hunk 1]
```diff
@@ -414,6 +414,45 @@
...
+    "X-Tyk-MockResponse": {
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "type": "boolean"
+        },
+        "code": {
+          "type": "integer"
+        },
+        "contentType": {
```
Comment (jeffy-mathew) [inline apidef/oas/schema/x-tyk-api-gateway.json:426]: we don't have `contentType` in `MockResponse` right? 
Also `body` is missing

**Review by jeffy-mathew — APPROVED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4324/merge
Commit: 50f7b1f35e46bf0ae994d5872befacca69f39a60 
Triggered by: pull_request (@furkansenharputlu)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3182191202)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4324/merge
Commit: f833dc436ff26e147ab51d886fc9690c4c7fcbe6 
Triggered by: pull_request (@furkansenharputlu)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3182215131)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/heads/master
Commit: b22caf9105b261b4f2bc4b8cdf3ea0e7cb8cc891 Add OAS mock response schema (#4324)
Triggered by: push (@furkansenharputlu)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3182315650)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4324/merge
Commit:  
Triggered by: pull_request (@furkansenharputlu)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3182180500)

**Pull request merged**